src/hearings/containers/hearing-actuals/hearing-actuals-view-edit-parties/hearing-actuals-view-edit-parties.component.ts

```diff
@@ -186,41 +185,45 @@ export class HearingActualsViewEditPartiesComponent implements OnInit, OnDestroy
-      });
+    if (actualParty?.individualDetails) {
+      if (this.isPlannedParty(actualParty)) {
+        this.participants.push({
+          name: `${actualParty.individualDetails.firstName} ${actualParty.individualDetails.lastName}`,
+          id: actualParty.actualPartyId
+        });
+      }
+      this.parties.push(this.fb.group({
+        firstName: [actualParty.individualDetails.firstName, [this.validators.mandatory('Enter first name')]],
+        lastName: [actualParty.individualDetails.lastName, this.isPlannedParty(actualParty) ? [] : [this.validators.mandatory('Enter last name')]],
+        role: [actualParty.partyRole, this.isPlannedParty(actualParty) ? [] : [this.validators.mandatory('Enter party role')]],
+        attendanceType: [actualParty.partyChannelSubType, [this.validators.mandatory('Enter attendance type')]],
+        organisation: [actualParty.actualOrganisationName],
+        attendeeRepresenting: [actualParty.representedParty, this.isPlannedParty(actualParty) ? [] : [this.validators.mandatory('Enter attendee representing')]],
+        partyId: [actualParty.actualPartyId],
+        isPlannedParty: [this.isPlannedParty(actualParty)]
+      }));
     }
-    this.parties.push(this.fb.group({
-      firstName: [actualParty.individualDetails.firstName, [this.validators.mandatory('Enter first name')]],
-      lastName: [actualParty.individualDetails.lastName, this.isPlannedParty(actualParty) ? [] : [this.validators.mandatory('Enter last name')]],
-      role: [actualParty.partyRole, this.isPlannedParty(actualParty) ? [] : [this.validators.mandatory('Enter party role')]],
-      attendanceType: [actualParty.partyChannelSubType, [this.validators.mandatory('Enter attendance type')]],
-      organisation: [actualParty.actualOrganisationName],
-      attendeeRepresenting: [actualParty.representedParty, this.isPlannedParty(actualParty) ? [] : [this.validators.mandatory('Enter attendee representing')]],
-      partyId: [actualParty.actualPartyId],
-      isPlannedParty: [this.isPlannedParty(actualParty)]
-    }));
   }
 
   private addPlannedParty(plannedParty: PlannedDayPartyModel) {
-    if (plannedParty.partyID) {
-      this.participants.push({
-        name: `${plannedParty.individualDetails.firstName} ${plannedParty.individualDetails.lastName}`,
-        id: plannedParty.partyID
-      });
+    if (plannedParty?.individualDetails) {
+      if (plannedParty.partyID) {
+        this.participants.push({
+          name: `${plannedParty.individualDetails.firstName} ${plannedParty.individualDetails.lastName}`,
+          id: plannedParty.partyID
+        });
+      }
+      this.parties.push(this.fb.group({
+        firstName: [plannedParty.individualDetails.firstName, null],
+        lastName: [plannedParty.individualDetails.lastName, null],
+        role: [plannedParty.partyRole, null],
+        attendanceType: [plannedParty.partyChannelSubType, [this.validators.mandatory('Enter attendance type')]],
```

Comment:
@johnbenjamin-hmcts - same question about whether Welsh translation is required for the validator text

Reply:
@Daniel-Lam - I have not changed this line of code as part of this fix, just wrapped it around with an _if_ statement. Might be there is another ticket to handle the Welsh for Hearings, but I am not sure.
I will check with our BA regarding this.